projects/observability/src/pages/apis/service-detail/instrumentation/components/panel-content.component.ts

```diff
@@ -14,7 +18,9 @@ import { HeuristicScoreInfo } from '../service-instrumentation.types';
     <p class="metric" *ngIf="this.heuristicScore?.sampleIds.length > 0">
       <b>Example {{ this.heuristicScore?.sampleType }}s: </b>
       <span *ngFor="let exampleId of this.heuristicScore?.sampleIds; let i = index">
-        <a title="Open in Explorer" [href]="this.getExampleLink(exampleId)">{{ exampleId }}</a>
+        <a target="_blank" title="Open in Explorer" [href]="this.getExampleLink(exampleId)">{{
```

Comment:
Did we get a request to make the example link open in new tab? It's in-app navigation so it should default to same page. Users can always `right click > open in new tab` if they want but adding `target="_blank"` is forcing a new tab even if the users don't want it.